ReSharper.FSharp/src/FSharp.Psi.Services/src/Generate/GenerateProvider.fs

```diff
@@ -293,6 +295,11 @@ type FSharpOverridingMembersBuilder() =
                 if diff > 0 then shiftWithWhitespaceBefore -diff unionRepr
                 null, null
 
+            | :? ITypeAbbreviationRepresentation as abbrRepr ->
```

Comment:
I think it should be possible to combine it with the previous branch, so there's no code duplication added.

```fsharp
| :? IUnionRepresentation
| :? ITypeAbbreviationRepresentation as typeRepr ->
    ...
```

Reply:
Ups, yeah. Done.